resources/user-agents/browsers/opera-mobile/opera-mobile-14-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Opera Mobile #MAJORVER#.#MINORVER#",
-  "versions": [52, 51, 50, 49, 48, 47, 46, 45, 44, 43, 42, 41, 40, 39, 38, 37, 36, 35, 34, 33, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14],
+  "versions": [52, 51, 50, 49, 48, "47.2", "47.1", 47, "46.1", 46, 45, 44, 43, "42.8", "42.7", "42.6", "42.5", "42.4", "42.3", "42.2", "42.1", 42, "41.3", "41.2", "41.1", 41, 40, 39, 38, "37.22", "37.18", "37.15", "37.12", "37.3", "37.1", 37, "36.2", "36.1", 36, "35.1", 35, 34, 33, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, "20.1", 20, 19, 18, 17, 16, 15, 14],
```

Comment:
Any particular reason you want to add minor version tracking for this browser?

Reply:
Not really, but most Opera Uas fail in the comparison with other parsers because of the missing minor version.